Makefile.am

```diff
@@ -23,3 +23,7 @@ ACLOCAL_AMFLAGS = -Im4/autotools
 SUBDIRS = m4 include lib src doc man
 dist_doc_DATA = README.md
 dist_noinst_DATA = tlog.spec
+# Remove *.gcno files from all directories.
+clean-local:
+	find $(top_builddir) -name '*.gcno' -delete
+	find $(top_builddir) -name '*.gcda' -delete
```

Comment:
This is not the best approach and a proper one would be putting `clean-local` target into every Makefile.am which builds .c files instead, but perhaps this will do. Please quote `$(top_builddir)` with single quotes.

Lastly, this can be combined into a single command like this:

    find '$(top_builddir)' -name '*.gcno' -or -name '*.gcda' -delete